cloudevents/extensions/loglevel.md

```diff
@@ -0,0 +1,153 @@
+# Log Level Extension
+
+
+## Attributes
+
+### loglevelname 
+
+- Type: `String`
+- Description: Human readable representation of the log level
+- Constraints
+  - OPTIONAL
+  - if present, MUST be a non-empty string
+  - SHOULD be lowercase
+  - RECOMMENDED values include   `emergency`, `alert` , `critical` ,`error`, `warning`, `notice`, `info`, `debug`, and
+     `verbose` , but  others MAY be used. The recommended 
+  - SHOULD have a one-to-one relationship with a distinct `loglevelnum` in the scope of
+   the `source`
+
+### loglevelnum 
+
+- Type: `Integer`
+
+- Description: A numerical representation of the level of importance of the event. 
+    Conforms to the severity ordering specified by [RFC5424](https://www.rfc-editor.org/rfc/rfc5424.html#section-6.2.1): severity of all levels is
+     assumed to be numerically ascending from most important to least important.
+  This specification does not define "importance" or meaning of each value other than
+  an event with a lower numerical value MUST be of more importance than an event with
+  a lower numerical value in the same `source`.
+
+- Constraints
+  - OPTIONAL
+  - if present, MUST NOT be negative
+  - RECOMMENDED values include
+    - `0` for `emergency`
+    - `1` for `alert`
+    - `2` for `critical`
+    - `3` for `error`
+    - `4` for `warning`
+    - `5` for `notice`
+    - `6` for `info`
+    - `7` for `debug`
+    - `8` for `verbose`
+
+
+## RECOMMENDED Log Mappings
+
+### Syslog
+
+All syslog log records SHOULD assign the severity level to `loglevelnum`. In addition to that
+
+The corresponding `loglevelname`s to each of the severity values
+
+| Syslog Severity | `loglevelname` |
+| --------------- | -------------- |
+| 0               | `emergency`    |
+| 1               | `alert`        |
+| 2               | `critical`     |
+| 3               | `error`        |
+| 4               | `warning`      |
+| 5               | `notice`       |
+| 7               | `info`         |
+| 8               | `debug`        |
```

Comment:
feels a bit odd that these values don't align with the recommended values above

Reply:
oops